src/lib/formatReferenceNumber.ts

```diff
@@ -0,0 +1,9 @@
+export const formatReferenceNumber = (number: string) => {
```

Comment:
Bu fonksiyonu replace edebiliriz.
export const formatReferenceNumber = (number: string) => {
  return number.replace(/\B(?=(\d{3})+(?!\d))/g, ' ')
}

bu daha az veya daha fazla rakamlı olan herhangi bir input geldiğinde 3'erli olarak ayırır. 10 haneden daha fazla veya daha az olabilecek durumlar için ona göre düzenleme yapsak daha kullanışlı bir kod oluşturmuş oluruz.